src/session/content/send_photo_dialog.rs

```diff
@@ -59,6 +60,27 @@ mod imp {
         fn constructed(&self, obj: &Self::Type) {
             self.parent_constructed(obj);
 
+            // Handle the enter key to send the message and also the combination of if with the
+            // right modifier keys to add new lines to the entry
+            let key_events = gtk::EventControllerKey::new();
+            self.caption_entry.add_text_view_controller(&key_events);
```

Comment:
Since we always want this controller for the MessageEntry I think it makes sense integrating this inside the MessageEntry itself. We can also expose the "activation" via a custom "activate" signal.